docs/run/nodes/nodes.md

```diff
@@ -16,9 +16,7 @@ At this time, any application that wishes to deploy to `mainnet` needs to run th
 
 **Ceramic networks** – There are currently three Ceramic networks: `mainnet`, `testnet-clay`, and `dev-unstable`. Learn more about each network [here](https://developers.ceramic.network/learn/networks/). By default, Ceramic will connect to `testnet-clay` and a [Ceramic Anchor Service](https://github.com/ceramicnetwork/ceramic-anchor-service) running on Ethereum Ropsten. When you are ready to get on Ceramic `mainnet`, reach out on the [Ceramic Discord →](https://chat.ceramic.network) to get access to our `mainnet` anchor service.
 
-**Running IPFS** – Ceramic relies on a system called [IPFS](https://docs.ipfs.io/) to connect to and share data in Ceramic networks. IPFS can run in a "bundled" mode alongside the Ceramic daemon or in "unbundled" mode, in which Ceramic connects to IPFS over HTTP. **Bundled** mode is designed for testing and local development only. **Unbundled** mode is designed for production and accommodates both local and cloud setups. It allows for more configuration options, controlled resource allocation, maintenance, debugging, and observability. (Note that Ceramic only works with `go-ipfs` version 0.11 or later.)
-
-The rest of this guide assumes you are running Ceramic with IPFS unbundled.
+**Running IPFS** – Ceramic relies on a system called [IPFS](https://docs.ipfs.io/) to connect to and share data in Ceramic networks. IPFS runs as a separate process from the Ceramic node itself, with each Ceramic node connected to a dedicated IPFS node over HTTP. The Ceramic Daemon can launch an IPFS process automatically (referred to as running ipfs in "bundled" mode in the Ceramic config file), which can be helpful for getting started quickly for testing and local development, but for production deployments we recommend running your own IPFS process manually and pointing your Ceramic node at it (referred to as running ipfs in "remote" mode in the Ceramic config file). This allows for more configuration options for your IPFS node allowing for more controlled resource allocation, as well as improved maintenance, debugging and observability. Note that Ceramic only supports `go-ipfs` version 0.12 or later.
```

Comment:
Would like to revert this language, because "bundled" mode does not work for production setups.

So I think it's better to say "is designed for testing and local development only" instead of making a recommendation otherwise. (We discussed this awhile ago and @ukstv confirmed but I don't remember the precise reason)

Reply:
reworded a bit - better?

Reply:
BTW, current wording is fine by me.